packages/core/chrome/core-chrome-browser-internal/src/chrome_service.tsx

```diff
@@ -240,7 +240,8 @@ export class ChromeService {
   }: StartDeps): Promise<InternalChromeStart> {
     this.initVisibility(application);
     this.handleEuiFullScreenChanges();
-    this.handleEuiDevProviderWarning(notifications);
+    // commented out until https://github.com/elastic/kibana/issues/201805 can be fixed
+    // this.handleEuiDevProviderWarning(notifications);
```

Comment:
@shahzad31 it looks like this creates a TS warning that might cause a CI failure, since `handleEuiDevProviderWarning` is declared but not used.

It looks like the warning is disregarded if we need to add `// @ts-expect-error` above line 185